tests/cypress/forms/forms_spec.js

```diff
@@ -210,6 +210,43 @@ context("Interactive marketo forms", () => {
       cy.url().should("include", "#success");
     }
   );
+
+  // wrote separate test for /kubernetes and /kubernetes/managed page as cypress couldn't find the job title input field by label text
```

Comment:
Not in scope of this PR, but if the issue with job title not being found by text is recurring I would look at maybe as a rule just adding a data-testid to all job title fields and then these forms can be included in the array with the others.